### PR TITLE
Document on how to manage log files

### PR DESCRIPTION
**Pull request by vishesh92**
related discussion: https://github.com/apache/cloudstack/pull/9849

----
📚 Documentation preview 📚: https://cloudstack-documentation--450.org.readthedocs.build/en/450/

### CODE REVIEW

**Comment by borisstoyanov** [comment]
@blueorangutan docbuild

File: source/adminguide/management.rst [hunk 1]
```diff
@@ -517,6 +517,29 @@ rules.
...
+directory and is named `log4j-cloud.xml`.
+
+By default, cloudstack uses `TimeBasedTriggeringPolicy` which rolls over
+the log file every day and are kept indefinitely. The log files are 
+compressed and archived in the same directory.
+
+Over time, the logs can fill up the entire disk space. To avoid this, you can 
+update the log4j-cloud.xml file to change the log file rollover and retention 
+policy. You can checkout some configuration recipes from the log4j2 documentation
+`here <https://logging.apache.org/log4j/2.x/manual/appenders/rolling-file.html#recipes>`_.
```
[the remark this answers is not in the document]
Comment (vishesh92) [reply source/adminguide/management.rst:541]: Should we provide the entire config file or a sample diff of the change?

[the remark this answers is not in the document]
Comment (vishesh92) [reply source/adminguide/management.rst:564]: I have added a sample diff to show how the change can be done.

**Comment by vishesh92** [comment]
@blueorangutan docbuild

**Review by rohityadavcloud — APPROVED** [review]

**Pull request merged**